Update reports.adoc (#109)
mulesoft/docs-monitoring: modules/ROOT/pages/reports.adoc

+
To view other reports, click the corresponding tab for the report to view. From any report tab, you can click the app name to view statistics for that app on its corresponding builtin dashboard.

[NOTE]

> For applications running on stand-alone runtime i.e., Hybrid environment, Datas are shown only for Requests, Performance and Failures tabs. CPU and Memory Utilization data are not shown for Hybrid applications.

Reviewer: ```suggestion
For applications that run using a stand-alone runtime or hybrid environments, data is shown only for Requests, Performance, and Failures tabs. CPU and memory utilization data are not shown for hybrid applications.
```
